### DISCUSSION TITLE
Fail Safe enabled

### DISCUSSION THREAD

**Discussion by kwanyongcho7** [microsoft/AirSim#3554] [Support Q&A] [answered]
Is there any way I can receive fail safe information using python api? 

I am using px4 airsim and sometimes, I got Failsafe Enabled : no global position or no local position.

Can I get this info through python api?

**Comment by jonyMarino** [comment, marked as answer]
Hi @kwanyongcho7! AirSim doesn't have an API for that. But we are working on that issue with px4.
